src/shared/components/ui/Calendar/CalendarHeader.tsx

```diff
@@ -0,0 +1,27 @@
+import { Button, Heading } from "react-aria-components";
+import { tv } from "tailwind-variants";
+
+const calendarHeader = tv({
+  slots: {
+    root: "flex justify-between pb-3",
+    previous: "text-gray-400",
+    next: "text-gray-400",
+    heading: "font-semibold first-letter:uppercase",
+  },
+});
+
+const { root, previous, next, heading } = calendarHeader();
```

Comment:
The Rood and Heading doesn't confuse me, the previous and next yes. They seem to be a function for an onClick but they are the classes. Can we rename these two (perhaps the four of them for consistency) to be called something like getPreviousClassName or similar please?